### PR TITLE
Update render settings.

### PR DESCRIPTION
**Pull request by siarheishut**
Add field layers in RenderingSettings and update MapRenderer accordingly.

### CODE REVIEW

**Comment by siarheishut** [comment]
I'm not sure about is it a good practice to create functions in anonymous namespace and pass the lvalue ref of object, like I did in map_renderer.cpp. Can I rewrite it better?

File: src/map_renderer.cpp [hunk 1]
```diff
@@ -97,70 +196,15 @@ MapRenderer::MapRenderer(
...
-                           .SetOffset(settings.stop_label_offset)
-                           .SetFontSize(settings.stop_label_font_size)
-                           .SetFontFamily("Verdana")
-                           .SetData(std::string(stop))
-                           .SetFillColor("black")));
+  for (auto &layer : settings.layers) {
+    if (layer == "bus_lines")
+      BusRoutesLayout(*this, buses, stops, settings, converter);
+    else if (layer == "bus_labels")
+      BusNamesLayout(*this, buses, stops, settings, converter);
```
Comment (foxfrikses) [inline src/map_renderer.cpp:203]: These should be methods

Comment (siarheishut) [reply src/map_renderer.cpp]: Will fix in the next PR.

File: tests/settings_parser_test.cpp [hunk 2]
```diff
@@ -81,7 +81,8 @@ TEST(TestParseSettings, TestRedneringSettings) {
                                         "purple", json::List{15, 23, 41},
                                         json::List{210, 81, 14, 0.94}}},
                                     {"bus_label_font_size", 6},
-                                    {"bus_label_offset", json::List{5, -7}}};
+                                    {"bus_label_offset", json::List{5, -7}},
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply tests/settings_parser_test.cpp:84]: Done.

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**